cgi-bin/DW/Media.pm

```diff
@@ -202,8 +202,17 @@ sub get_active_for_user {
     return () unless $rows && ref $rows eq 'ARRAY';
 
     # construct media objects for each of the items and return that
-    return sort { $b->logtime <=> $a->logtime }
-           map { DW::Media->new( user => $u, mediaid => $_, %opts ) } @$rows;
+    my @media;
+    foreach ( @$rows ) {
+        # use eval to catch croaks
+        my $obj = eval { DW::Media->new( user => $u, mediaid => $_, %opts ) };
```

Comment:
I sort of wonder what the best way to actually handle errors in Perl is. If the library hits a problem, croak? And let upstreams catch it? Or return undefs and errstrs? I don't even know anymore and we use every single paradigm in the DW code...